client/src/components/RecordsUtils/RecordTable.js

```diff
@@ -32,13 +32,17 @@ export default function RecordTable(props) {
             .toLowerCase()
             .includes(props.inputVal) ||
           record.asset_tag.toString().toLowerCase().includes(props.inputVal.toLowerCase())
+          
       );
     }
-
+    //Filters Records table by only checked out items
     if (props.filterByCheckedOut) {
       allRecords = allRecords.filter((record) => record.in_date === null);
     }
-
+    //Filters Records Table by range of selected dates
+    if(props.selectedDate1 || props.selectedDate2  ){
+      allRecords = allRecords.filter((record) => record.due_date >= props.selectedDate1 && record.due_date <= props.selectedDate2);
```

Comment:
Can we make it so that either one can be selected? Currently you have to select both in order to specify a specific range, but can we instead just say all from this day, or up to this day?